tiledb/sm/query/result_tile.cc

```diff
@@ -37,8 +37,10 @@
 #include "tiledb/sm/fragment/fragment_metadata.h"
 
 #include <cassert>
+#include <chrono>
 #include <iostream>
 #include <list>
+#include <thread>
```

Comment:
Are those needed?

Reply:
not necessary. I added it for debugging. will remove it